lib/plugins/aws/deploy/lib/initializeResources.js

```diff
@@ -45,17 +44,13 @@ module.exports = {
         .Statement.concat(this.serverless.service.provider.iamRoleStatements);
     }
 
+    this.serverless.service.custom = {};
```

Comment:
any particular reason why this object is nested in the `serverless.service` ? can't it just be in the outer `this` context? my worry is that we may introduce a `custom` property in `serverless.yml` in the future which will conflict here. Since you're just storing/passing values in memory, I'd just keep it out of the `service` (and even `serverless`) objects, and keep this `service` object a reflection of `serverless.yml`.